src/docs-components/CornerDialogDocs.tsx

```diff
@@ -0,0 +1,55 @@
+import { capitalize } from 'lodash';
+
+import { SectionHeader } from './common/SectionHeader';
+
+import { CornerDialog } from '@/components/CornerDialog';
+import { tet } from '@/tetrisly';
+
+const intents = ['none', 'warning', 'negative'] as const;
+
+export const CornerDialogDocs = () => (
+  <tet.section display="flex" padding="40px 0px" flexDirection="column">
```

Comment:
```suggestion
  <tet.section display="flex" padding="$dimension-500 0px" flexDirection="column">
```

Reply:
Should I use also `$dimension-0` token to be consistent?

```suggestion
  <tet.section display="flex" padding="padding="$dimension-500 $dimension-0" flexDirection="column">
```

Reply:
Sure, you can use it ;)